about.html

```diff
@@ -13,52 +13,78 @@
+              <div class="about-boxes-item">
+          <img src="img/about-plan.png" alt="strategy">
+          <h2>Strategey</h2>
+
+              </p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+
+<div class="learn-more-button">Learn More</div>
+              </div>
+<div class="about-boxes-item">
+    <img src="img/about-working.png" alt="strategy">
     
-    Strategy
+        
+      <h2>How we work</h2>
+    
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
 
 
+        <div class="learn-more-button">Learn More</div>
+        </div>
+</div>
 
-    <img src="img/about-working.png" alt="strategy">
+<div class="about-boxes-row">
+<div class="about-boxes-item">
+  <img src="img/about-office.png" alt="strategy">
     
-    How We Work
+<h2>Places we work</h2>   
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
+  
+</p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
     
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
 
+<div class="learn-more-button">Learn More</div>
+  </div>
+<div class="about-boxes-item">
     <img src="img/about-meeting.png" alt="strategy">
+     <h2>Collaboration</h2>
     
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    Let's Work Together
+        <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-        
+      <div class="learn-more-button">Learn More</div>
+     </div>
+     </div>
+     </section>    
+     
+   <section class="work-together">
+    <h2>Let's Work Together</h2>
+   </p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+  </section>      
       <footer>
         <nav>
           <a href="index.html">Home</a>
```

Comment:
Please try to indent your code and add whitespace to make your code cleaner and easier to read.